proto/handle.proto

```diff
@@ -28,14 +28,14 @@ extend google.protobuf.FieldOptions {
 
 // Sender handle for an Oak channel.
 // This type is sent over the wire, but in types generated by prost it is replaced with
-// [`oak::io::Sender`](../io/struct.Sender.html).
+// `oak::io::Sender`.
```

Comment:
So I thought the whole point was to make this a standard link like ``[`oak::io::Sender`]``, why does that not work now?

Reply:
It cannot resolve it:
```
 --> /opt/my-project/oak_io/target/debug/build/oak_io-4bc20a416904bf4a/out/oak.handle.rs:3:6
  |
3 | /// [`oak::io::Sender`].
  |      ^^^^^^^^^^^^^^^^^ cannot be resolved, ignoring
```

Reply:
`oak_io` doesn't reference the SDK, so it cannot be resolved even though the SDK re-exports the types. In `oak_io` it should be able to resolve something like `crate::Sender` or `super::Sender`. I don't know whether it would cause problems in the cases where handle.proto is imported in other proto files compiled in other crates.

Reply:
`crate::Sender` works fine, but there are problems, if we want to use `GrpcInvocation` in one of the examples:
```
Found invalid urls in /opt/my-project/examples/target/doc/trusted_database_handler/proto/oak/invocation/index.html:
	Linked file at path /opt/my-project/examples/target/doc/trusted_database_handler/proto/oak/invocation/oak_io::Receiver does not exist!
	Linked file at path /opt/my-project/examples/target/doc/trusted_database_handler/proto/oak/invocation/oak_io::Sender does not exist!
```